.github/CODEOWNERS

```
@@ -3,45 +3,45 @@
 # These are suggestions; it is fine to have your changes reviewed by someone
 # else.
 
-.github/                        @dart-ecosystem-team
-README.md                       @dart-ecosystem-team
-pkgs/bazel_worker               @dart-bat
-pkgs/benchmark_harness          @dart-native-runtime-team
-pkgs/boolean_selector           @dart-ecosystem-team
-pkgs/browser_launcher           @dart-ecosystem-team
+.github/                        @dart-lang/dart-ecosystem-team
+README.md                       @dart-lang/dart-ecosystem-team
+pkgs/bazel_worker               @dart-lang/dart-bat
+pkgs/benchmark_harness          @dart-lang/dart-native-runtime-team
+pkgs/boolean_selector           @dart-lang/dart-ecosystem-team
+pkgs/browser_launcher           @dart-lang/dart-ecosystem-team
 pkgs/cli_config                 @dcharkes
 pkgs/cli_util                   @devoncarew
-pkgs/clock                      @dart-ecosystem-team
-pkgs/code_builder               @dart-ecosystem-team
-pkgs/coverage                   @liamappelbe @dart-native-runtime-team
-pkgs/csslib                     @dart-ecosystem-team #formerly @leonsenft
-pkgs/extension_discovery        @dart-ecosystem-team #tbd
-pkgs/file                       @dart-ecosystem-team
-pkgs/file_testing               @dart-ecosystem-team
-pkgs/glob                       @dart-ecosystem-team
+pkgs/clock                      @dart-lang/dart-ecosystem-team
+pkgs/code_builder               @dart-lang/dart-ecosystem-team
+pkgs/coverage                   @liamappelbe @dart-lang/dart-native-runtime-team
+pkgs/csslib                     @dart-lang/dart-ecosystem-team #formerly @leonsenft
+pkgs/extension_discovery        @dart-lang/dart-ecosystem-team #tbd
+pkgs/file                       @dart-lang/dart-ecosystem-team
+pkgs/file_testing               @dart-lang/dart-ecosystem-team
+pkgs/glob                       @dart-lang/dart-ecosystem-team
 pkgs/graphs                     @natebosch
 pkgs/html                       @HosseinYousefi
-pkgs/io                         @dart-bat
-pkgs/json_rpc_2                 @dart-ecosystem-team
-pkgs/markdown                   @analyzer-team
+pkgs/io                         @dart-lang/dart-bat
+pkgs/json_rpc_2                 @dart-lang/dart-ecosystem-team
+pkgs/markdown                   @dart-lang/dart-model-team @dart-lang/dash-developer-services
```

```suggestion
pkgs/markdown                   @srawlins @dart-lang/dart-model-team @dart-lang/dash-developer-services
```

And probably `dash-developer-experience` as a team logically, through I don't know that that's a github team.